src/DBus.vala

```diff
@@ -91,6 +91,15 @@ public class Notifications.Server : Object {
             app_info = new DesktopAppInfo (desktop_id);
         }
 
+        string? image_path = null;
+        if ((variant = hints.lookup ("image-path")) != null || (variant = hints.lookup ("image_path")) != null) {
+            image_path = variant.get_string ();
+
+            if (!image_path.has_prefix ("/") && !image_path.has_prefix ("file://")) {
```

Comment:
It seems like sometimes apps might try to pass in their icon name here and we don't want that